e2e/pages/EnableLocation.po.js

```diff
@@ -0,0 +1,19 @@
+const buttonlabel = 'Enable Location';
+const screenshotText = 'Enable Location Page';
+const screenShotWithMenuText = 'Location Permissions Dialog';
+
+class EnableLocation {
+  tapButton = async () => {
```

Comment:
these don't need `this` binding so normal form is fine

```js
async tapButton() {}
```

Reply:
Yup, cruft from a previous approach. will remove